solidity/contracts/converter/BancorConverterRegistry.sol

```diff
@@ -388,16 +387,9 @@ contract BancorConverterRegistry is IBancorConverterRegistry, ContractRegistryCl
     function isConverterValid(IBancorConverter _converter) public view returns (bool) {
         // verify the the smart token has a supply and that the converter is active
         ISmartToken token = ISmartTokenController(_converter).token();
-        if (token.totalSupply() == 0 || token.owner() != address(_converter))
+        if (token.owner() != address(_converter))

```

Comment:
@yudilevi : Well, the entire function can be reduced to `return token.owner() == address(_converter);`.

Reply:
Isn't it the case? Do you mean one liner to get the token and compare?

Reply:
@yudilevi : There's an `if`/`else` here. I say - let's go for a one-liner function.